frontend/src/container/NotificationWall.jsx

```diff
@@ -95,7 +95,7 @@ export class NotificationWall extends React.Component {
   }
 
   getNotificationDetails = notification => {
-    const { props } = this
+    const { props, state } = this
```

Comment:
Just to understand, did you added `state` for consistency?
Because you don't seem to use it

Reply:
If I'm not using it I probably just forgot to remove it, thanks!